.gitignore

```diff
@@ -0,0 +1,8 @@
+# .gitignore
+node_modules/
+
+<<<<<<< HEAD
+test.md
```

Comment:
- [OPTIONAL] In the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) it is mentioned that the `test.md` should not be in the Github repository.
You can achieve this by removing the file manually and committing your work once again.